### PR TITLE
Refine docker ci/cd

### PR DESCRIPTION
**Pull request by kacurez**
- **Build from php:~5.6~7.1** - instead of  keboola-base-php56 image
- **Test and deploy to ecr** - via travis
- **Developer portal is NOT UPDATED!!** - still using dockerhub image `keboola/tde-exporter`
- **Build docker image from this repository** - former dockerfile repo(https://github.com/keboola/docker-tde-exporter) can be removed as well as dockerhub image `keboola/tde-exporter` ONLY AFTER REPOSITORY SWITCH IN DEVELOPER PORTAL!!
- **Use docker-compose and makefile** - better developement and ci/cd workflow
- **Better testing** python test, php test and functional test. Run these tests separately in a container and not during the build.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+FROM php:5.6
```
Comment (Halama) [inline Dockerfile]: proč ne 7.1?

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+docker pull quay.io/keboola/developer-portal-cli-v2:latest
```
Comment (Halama) [inline deploy.sh]: Nemáš nastavený exit při chybě!

File: Dockerfile (hunk 1 above)
Comment (kacurez) [reply Dockerfile]: nechcel som to robit v tomto PR, chcem aby sa to co najviac podobalo sucasnemu "runtime" prostrediu

File: deploy.sh (hunk 2 above)
Comment (kacurez) [reply deploy.sh]: nerozumiem, myslis ten hashbang?

Comment (Halama) [reply deploy.sh]: no ted se ti spustej vsechny prikazy i kdyz prvni treba failne.

Comment (Halama) [reply deploy.sh]: chybí tam tohle https://github.com/keboola/darksky-augmentation/blob/58938953b8eff710c297d911b874a6e13338fb83/deploy.sh#L2

Comment (kacurez) [reply deploy.sh]: aha jasne, ja som to zkopcil odtialto:
https://github.com/keboola/ex-snowflake-query-history/blob/master/deploy.sh
pridam to tam dik!

File: Dockerfile (hunk 1 above)
Comment (kacurez) [reply Dockerfile]: skusil som to s 7.1 a testy presli tak to teda dam na 7.1

**Pull request merged**